pkg/transport/transport.go

```diff
@@ -64,9 +64,11 @@ func (transport *Transport) DialNode(ctx context.Context, node *pb.Node, opts ..
 		return nil, Error.New("no address")
 	}
 
-	// add ID of node we are wanting to connect to
-	dialOpt := transport.tlsOpts.DialOption(node.Id)
-	options := append([]grpc.DialOption{dialOpt, grpc.WithBlock(), grpc.FailOnNonTempDialError(true)}, opts...)
+	options := append([]grpc.DialOption{
+		transport.tlsOpts.DialOption(ctx, node.Id),
+		grpc.WithBlock(),
+		grpc.FailOnNonTempDialError(true),
```

Comment:
should this still be here?

Reply:
Yes

Reply:
oh, sorry, for some reason i thought it was FailOnTempDialError